graphics/src/MeshManager.cc

```diff
@@ -16,11 +16,23 @@
  */
 
 #include <sys/stat.h>
-#include <string>
+
+#include <cctype>
+#include <cstdint>
 #include <mutex>
+#include <string>
 #include <unordered_map>
 #include <unordered_set>
-#include <cctype>
+#include <vector>
+
+// Suppress warnings for VHACD
+#pragma GCC diagnostic push
```

Comment:
This is generating a windows warning.

Reply:
fxied in [d92687c](https://github.com/gazebosim/gz-common/pull/585/commits/d92687c4f150f2cdee48f21ff34bd0f6a884ee0b)